### PR TITLE
benches: fix resource download and extract.

### PR DESCRIPTION
**Pull request by hicqu**

### CODE REVIEW

**Comment by hicqu** [comment]
@BusyJay @AndreMouche , PTAL.

**Comment by hicqu** [comment]
@BusyJay , PTAL, thanks.

File: benches/coprocessor/codec/mysql/json/mod.rs [hunk 1]
```diff
@@ -48,15 +48,15 @@ fn download_and_extract_file(url: &str) -> io::Result<String> {
...
     try!(th.join().unwrap());
     assert_eq!(output.status.code(), Some(0));
     Ok(String::from_utf8(output.stdout).unwrap())
 }
 
 pub fn load_test_jsons() -> Vec<String> {
-    let url = "https://download.pingcap.org/resources/world_bank.json.bz2";
+    let url = "https://download.pingcap.org/resources/world_bank.json.tar.gz";
     download_and_extract_file(url)
         .unwrap()
```
Comment (overvenus) [inline benches/coprocessor/codec/mysql/json/mod.rs]: I think we should return error to caller, let caller to decide to `unwrap()` or just simply skip benchmark.

**Comment by BusyJay** [comment]
Let's add a `#[ignore]` to the test cases for now. We generally only want to bench it rather than test it.

**Comment by hicqu** [comment]
Now we can use `cargo bench --features "default" -- --nocapture --ignored` to run those benches.

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**